README.rst

```diff
@@ -27,7 +26,7 @@ Run locally
 ===========
 
 Kinto is based on top of the `cliquet <https://cliquet.rtfd.org>`_ project, and
-as such, please follow the installation instructions there to get started.
+as such, please refer to its documentation regarding API and endpoints.
```

Comment:
``refer to cliquet's documentation`` even if its is right, it is easier to understand I think.